src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+@mixin center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+@keyframes rotate-block {
+  from {
+    transform: translate(-50%, -100%) rotate(0deg);
+  }
+  to {
+    transform: translate(-50%, -100%) rotate(360deg);
+  }
+}
+
 body {
   margin: 0;
```

Comment:
In the `body` selector, it's good practice to also set a default font-family, unless it's being inherited from a global style elsewhere. This can help maintain consistency across browsers.